### PR TITLE
Customizable Quick Pack Switch key

### PR DESCRIPTION
**Pull request by IvoryDuke**
Assigned Focus bind to control the quick switch function when in menu (its in game behavior is unchanged).
Also:
- replaced a #define in Config.cpp with a constexpr;
- added pack change keys to RefreshBinds();
- small tweak in formatLevelDescription().

A new custom bind can be made but that would make the bind customization menu go out of bounds and would require a bigger amount of effort.

### CODE REVIEW

**Comment by IvoryDuke** [comment]
Working on making it work for controller too.

File: src/SSVOpenHexagon/Core/Joystick.cpp [hunk 1]
```diff
@@ -134,6 +134,11 @@ enum class AxisDir : int
     Right = 1
 };
 
+inline AxisDir operator-(AxisDir dir)
```
Comment (SuperV1234) [inline src/SSVOpenHexagon/Core/Joystick.cpp:137]:
```suggestion
[[nodiscard]] AxisDir operator-(AxisDir dir)
```

No need for `inline` here, TBH.

File: src/SSVOpenHexagon/Core/MenuGame.cpp [hunk 2]
```diff
@@ -1686,6 +1688,16 @@ void MenuGame::update(ssvu::FT mFT)
...
 
+    if(hg::Joystick::nextPackRisingEdge())
+    {
+        changePackAction(1);
+    }
+    else if(hg::Joystick::previousPackRisingEdge())
+    {
+        changePackAction(-1);
+    }
+    focusHeld = hg::Joystick::focusPressed();
```
Comment (SuperV1234) [inline src/SSVOpenHexagon/Core/MenuGame.cpp:1699]:
```suggestion
    }
    
    focusHeld = hg::Joystick::focusPressed();
```

I like vertical space :)

**Review by SuperV1234 — COMMENTED** [review]
Looks good so far. Waiting for the controller changes before merging it in!

**Comment by IvoryDuke** [comment]
In case it was not clear: controller is fully working

**Review by SuperV1234 — APPROVED** [review]

**Pull request merged**